apps/designer/app/designer/features/workspace/canvas-tools/text-toolbar/text-toolbar.tsx

```diff
@@ -2,11 +2,26 @@ import { type Publish } from "~/shared/pubsub";
 import { useMemo, useState, type MouseEventHandler } from "react";
 import {
   useSelectedInstanceData,
-  useSelectionRect,
+  type TextToolbarValue,
+  useTextToolbar,
 } from "~/designer/shared/nano-states";
 import { ToggleGroup, type CSS } from "@webstudio-is/design-system";
 import { FontBoldIcon, FontItalicIcon, Link2Icon } from "@webstudio-is/icons";
-import { createInstance } from "~/shared/tree-utils";
+import { useSubscribe } from "~/shared/pubsub";
+
+declare module "~/shared/pubsub" {
+  export interface PubsubMap {
+    textToolbarShow: TextToolbarValue;
```

Comment:
convention for names we used so far "verb in present tense": `showTextToolbar`, I think you started using redux action type convention:

Reply:
Just wanted prefix everything. But verb works too, will fix